lib/event_bg_bus.rb

```diff
@@ -40,6 +40,15 @@ def is_string_or_symbol?(s)
       s.class == String || s.class == Symbol
     end
 
+    def redis_safe_hash(hash)
+      out = { }
+      hash.each do |key, value|
+        out[key.to_s] = value.nil? ? '' : value
```

Comment:
Nil to empty string could this be unexpected?

Reply:
I had some reference somewhere that that is what was happening with previous versions of sidekiq/redis. I can't find it anymore.

Reply:
The new underlying redis client does not allow `null` so we need to do something by default. If the calling/reading code doesn't want `""` then it can provide  something that makes more sense to it.

Reply:
Pretty sure [this is the code](https://github.com/redis/redis-rb/blob/5aa1e0c5bf7c0c7f5410680a11ee19a689afbc17/lib/redis/connection/command_helper.rb#L7) that is currently being used to create the command sent to redis by sidekiq.
As you can see it calls `.to_s` on every thing that isn't an array, and `nil.to_s == ""`.